blockdag/dag.go

```diff
@@ -712,9 +713,10 @@ func (dag *BlockDAG) saveChangesFromBlock(block *util.Block, virtualUTXODiff *UT
 
 	// Update DAG state.
 	state := &dagState{
-		TipHashes:         dag.TipHashes(),
-		LastFinalityPoint: dag.lastFinalityPoint.hash,
-		LocalSubnetworkID: dag.subnetworkID,
+		TipHashes:               dag.TipHashes(),
+		LastFinalityPoint:       dag.lastFinalityPoint.hash,
+		ReachabilityReindexRoot: dag.reachabilityTree.reindexRoot.blockNode.hash,
```

Comment:
Pls put ReachabilityReindexRoot somewhere else and fetch it from db when you init reachabilityTree